Licensing_Program/utils/load_config.py

```diff
@@ -54,7 +54,7 @@ def validate_config_stream(stream):
     return valid_config
 
 
-def load_config(proj_dir):
+def load_config(proj_dir="."):
```

Comment:
How then will a user specify a project path, `proj_dir`, different from the current working directory.

In other words, is there a CLI option to override this variable?

Thereby, I could call `utils.load_config(path)` vs `utils.load_config(".")`